enterprise-suite/templates/alertmanager.yaml

```diff
@@ -111,9 +112,9 @@ metadata:
   name: expose-alertmanager
```

Comment:
I think we could just delete this service. We shouldn't need these expose services anymore.

Reply:
Yup, done this in a separate PR: https://github.com/lightbend/helm-charts/pull/197